libs/wingsdk/test/target-sim/__snapshots__/bucket.test.ts.snap

```diff
@@ -17,18 +17,21 @@ exports[`bucket on event creates 3 topics, and sends the right event and key in
   "Publish (message=a).",
   "Sending message (message=a, subscriber=sim-6).",
   "Put (key=a).",
+  "Bundled code (697326 bytes).",
```

Comment:
On one hand this info feels useful, though I think it might cause a lot more noisy snapshots... maybe I'll hide behind the DEBUG flag